src/Exception/Database/DatabaseExceptionTrait.php

```diff
@@ -7,7 +7,7 @@
     /** @var string */
     protected $driver;
 
-    private static $driverNames = [
+    protected static $driverNames = [
```

Comment:
When I do a stacktrace I see that `getPlatform` is called from:
`Bolt\Exception\Database\DatabaseException->getPlatform() called at [/var/www/hderuiter/incadat/vendor/bolt/bolt/src/EventListener/ExceptionListener.php:170]`